server/utils/dateUtils.ts

```diff
@@ -339,3 +339,12 @@ export const daysToWeeksAndDays = (days: string | number): { days: number; weeks
     weeks: durationWeeks,
   }
 }
+
```

Comment:
You could do this with regex - `^(2[0-3]|[01]?[0-9]):([0-5]?[0-9])$` and then it wouldn't need a try/catch

Reply:
You’re right, nice to avoid the try/catch. Though the second question mark probably isn’t needed, we do want 2 digits for the minutes!